### PR TITLE
fix: improve eks usage

### PR DESCRIPTION
**Pull request by diabhey**

### CODE REVIEW

**Review by j-zimnowoda — CHANGES_REQUESTED** [review]
I think that there should be a default `cluster_name` as this is quickstart

File: aws/eks/terraform.tfvars.example [hunk 1]
```diff
@@ -4,22 +4,34 @@
...
-aws_accounts = ""
+aws_accounts = ["222222222222"]
 
 # AWS IAM Roles
-aws_roles = ""
+aws_roles = [
+  {
+    rolearn  = "arn:aws:iam::222222222222:user1/username"
+    username = "username"
+    groups   = ["Admins"]
```
Comment (j-zimnowoda) [inline aws/eks/terraform.tfvars.example:14]: Are you sure that we need to provide groups?

File: aws/eks/terraform.tfvars.example [hunk 2]
```diff
@@ -4,22 +4,34 @@
...
+aws_users = [
+  {
+    rolearn  = "arn:aws:iam::222222222222:user1/username"
+    username = "username"
+    groups   = ["Admin"]
+  },
+]
+
+# EKS Cluster Name
+cluster_name = ""
```
Comment (j-zimnowoda) [inline aws/eks/terraform.tfvars.example:28]: why not providing default cluster_name?

File: aws/otomi-install/terraform.tfvars.example [hunk 3]
```diff
@@ -3,4 +3,14 @@
 # - Check variables.tf for variable type(includes defaults)
 # ==========================================================
 
-aws_region = ""
\ No newline at end of file
+# EKS Cluster Name
+cluster_name = ""
```
Comment (j-zimnowoda) [inline aws/otomi-install/terraform.tfvars.example:7]: Provide default name?

File: aws/eks/terraform.tfvars.example (hunk 2 above)
Comment (diabhey) [reply aws/eks/terraform.tfvars.example:28]: Agree. I will include it

File: aws/otomi-install/terraform.tfvars.example (hunk 3 above)
Comment (diabhey) [reply aws/otomi-install/terraform.tfvars.example:7]: Agree. I will include it

File: aws/eks/terraform.tfvars.example (hunk 1 above)
Comment (diabhey) [reply aws/eks/terraform.tfvars.example:14]: Yes, it is a mandatory field

**Review by j-zimnowoda — APPROVED** [review]

**Pull request merged**